tests/end-to-end/director/runtime_auth_test.go

```diff
@@ -0,0 +1,149 @@
+package director
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+
+	gcli "github.com/machinebox/graphql"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestQueryRuntimeAuths(t *testing.T) {
+	// GIVEN
+	ctx := context.Background()
+
+	defaultAuth := fixBasicAuth()
+	customAuth := graphql.AuthInput{
+		Credential: &graphql.CredentialDataInput{
+			Basic: &graphql.BasicCredentialDataInput{
+				Username: "custom",
+				Password: "auth",
+			}},
+		AdditionalHeaders: &graphql.HttpHeaders{
+			"customHeader": []string{"custom", "custom"},
+		},
+	}
+
+	exampleSaved := false
+
+	testCases := []struct {
+		Name string
+		Apis []*graphql.APIDefinitionInput
+	}{
+		{
+			Name: "API without default auth",
+			Apis: []*graphql.APIDefinitionInput{
+				{
+					Name:      "without-default-auth",
+					TargetURL: "http://mywordpress.com/comments",
+				},
+			},
+		},
+		{
+			Name: "API with default auth",
+			Apis: []*graphql.APIDefinitionInput{
+				{
+					Name:        "with-default-auth",
+					TargetURL:   "http://mywordpress.com/comments",
+					DefaultAuth: defaultAuth,
+				},
+			},
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.Name, func(t *testing.T) {
+			appInput := graphql.ApplicationInput{
+				Name: "test-app",
+				Apis: testCase.Apis,
+				Labels: &graphql.Labels{
+					"scenarios": []interface{}{"DEFAULT"},
+				},
+			}
+
+			app := createApplicationFromInputWithinTenant(t, ctx, appInput, defaultTenant)
+			defer deleteApplication(t, app.ID)
+
+			var rtmIDs []string
+			for i := 0; i < 10; i++ {
```

Comment:
You can create 3 runtime, because I am poor and ram is expensive nowadays :) 
![image](https://user-images.githubusercontent.com/45110612/64427171-d782b600-d0b0-11e9-9eec-c80b853d91f9.png)